### PR TITLE
Enumerate infrastructure inventory

### PR DESCRIPTION
**Pull request by jaimergp**
Closes https://github.com/Quansight-Labs/czi-conda-forge-mgmt/issues/32

### CODE REVIEW

File: docs/reference/feedstock-settings.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Feedstock settings
+
+When `conda-smithy` rerenders the contents of the feedstock, it takes into account different sources of configuration:
```
Comment (viniciusdc) [inline docs/reference/feedstock-settings.md:3]: > `conda-smithy`

Link to list of repos/tools

File: docs/reference/feedstock-settings.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+- `recipe/meta.yaml` (only the `skip: true` lines)
+- `.ci_support/migrations/*` (added by the autotick bot)
+
+:::caution
+Outside `recipe/`, the only file you are supposed to edit is `conda-forge.yml`.
+Everything else is automatically managed!
+:::
+
+## conda-forge.yml {#conda-forge-yml}
+
```
Comment (viniciusdc) [inline docs/reference/feedstock-settings.md:16]: As an observation, I think we should just go over bit-by-bit of the fields and what they generally mean (in a block fashion), and add more depth to it under an advanced page.... what do you think?

File: docs/reference/glossary.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
+# Glossary
+
```
Comment (viniciusdc) [inline docs/reference/glossary.md:2]: content delivery network (CDN) ?

File: docs/reference/infrastructure/feedstocks.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+- ⚙️ Deployed in Github repositories
+- 🔒 Has access to Azure Pipelines, Github Actions, Anaconda.org (cf-staging)
+- 🔐 Might have access to Travis CI, Cirun via `admin-requests` (WIP)
+- 🤖 Integrated with [`admin-migrations`](./automated-maintenance.md#admin-migrations), [`admin-requests`](./automated-maintenance.md#admin-requests), [`autotick-bot`](./automated-maintenance.md#autotick-bot), [`webservices`](./automated-maintenance.md#webservices).
+
+conda-forge has thousands of feedstocks.
+Each feedstock hosts a recipe plus the required pipelines, supporting scripts and configuration metadata.
+
+The contents of a feedstock are well specified. Only two locations are user-managed:
+
```
Comment (viniciusdc) [inline docs/reference/infrastructure/feedstocks.md:20]: maybe adding a general tree of contents for a feedstock here?
```
name
   \a
   \b...
```

Comment (viniciusdc) [reply docs/reference/infrastructure/feedstocks.md:20]: or at the end, to give a view of the organization of the repo?

File: docs/reference/infrastructure/services.md [hunk 5]
```diff
@@ -0,0 +1,174 @@
+# Services & providers
```
Comment (viniciusdc) [inline docs/reference/infrastructure/services.md:1]: I think we could subdivide this into categories, and everything that is GH based, anaconda related, organization related (teams, repos...) would be under its own group of [tabs ](https://docusaurus.io/docs/next/markdown-features/tabs)

File: docs/reference/feedstock-settings.md (hunk 1 above)
Comment (viniciusdc) [reply docs/reference/feedstock-settings.md:3]: link to its reference on [tooling-data.md](https://github.com/Quansight-Labs/cf-infra-docs/pull/2/files#diff-467cf5c6d44651073fb85ecdc393c674bdd2b2c80adebc71b7cda271bbe8ec23)

File: docs/reference/infrastructure/website.md [hunk 6]
```diff
@@ -0,0 +1,14 @@
+# Website
+
```
Comment (viniciusdc) [inline docs/reference/infrastructure/website.md:2]: image?

File: docs/reference/pinnings.md [hunk 7]
```diff
@@ -0,0 +1,135 @@
...
+
+For consistent builds and inter-artifact compatibility, many of the libraries and projects used to build packages on conda-forge are pinned or locked to a globally controlled value. We call this "pinnings". This includes compilers, runtime libraries, base systems, and others.
+
+:::info
+
+This information is collected here for your convenience, but it might not be up-to-date with the authoritative source: `conda-forge-pinnings`.
+[Check this file for up-to-date values](https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/main/recipe/conda_build_config.yaml).
+:::
+
+## Base systems
```
Comment (viniciusdc) [inline docs/reference/pinnings.md:15]: this would greatly benefit of using tabs :smile: (the tabs'guy ksksks)

File: docs/reference/runners.md [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+| [Azure Pipelines][azure] ([Microsoft-hosted][ms-hosted])  | `linux-64`, `win-64`, `linux-aarch64`†‡, `linux-ppc64le`†‡ |  2  | No  | 7GB  | 14GB SSD˟ |  6h   |       100       |          [More details][ms-hosted]           |
+|                             ↳                             |                   `osx-64`, `osx-arm64`‡                   |  3  |     | 14GB |           |       |                 |
+| [Github Actions][gh-actions] ([Github-hosted][gh-hosted]) | `linux-64`, `win-64`, `linux-aarch64`†‡, `linux-ppc64le`†‡ |  2  | No  | 7GB  | 14GB SSD˟ |  6h   |       100       | Reserved for internal conda-forge usage only |
+|                             ↳                             |                   `osx-64`, `osx-arm64`‡                   |  3  |     | 14GB |           |       |                 |
+|                    [Travis CI][travis]                    |              `linux-aarch64`, `linux-ppc64le`              |  2  | No  | ~4GB |   ~18GB   | 50min |        ?        |
```
Comment (viniciusdc) [inline docs/reference/runners.md:9]: is this up to date to what the actual CI's runners are currently using?

**Review by viniciusdc — COMMENTED** [review]
LGTM, I have some ideas for the overall presentation of most of those tables and indexes, but that can be added at a later moment. Outside of the aspect of linking some of the terms you mention during the text e.g conda-smith, conda-build tools, CDN, and artifacts... all look great!!! 

Also, it would be interesting if we could adopt [※] for the emulated symbols in the text references.

File: docs/reference/runners.md (hunk 8 above)
Comment (jaimergp) [reply docs/reference/runners.md:9]: Yea, mostly.

File: docs/reference/pinnings.md (hunk 7 above)
Comment (jaimergp) [reply docs/reference/pinnings.md:15]: I don't know how to do that in Docusaurus (yet) and for now I want to keep it simple, but feel free to create an issue!

File: docs/reference/infrastructure/website.md (hunk 6 above)
Comment (jaimergp) [reply docs/reference/infrastructure/website.md:2]: This is just the reference on how it's built, so I am afraid screenshots (if that's what you mean) would get outdated very quickly.

File: docs/reference/infrastructure/services.md (hunk 5 above)
Comment (jaimergp) [reply docs/reference/infrastructure/services.md:1]: Ah wait it does have tabs natively? Wow!

File: docs/reference/feedstock-settings.md (hunk 2 above)
Comment (jaimergp) [reply docs/reference/feedstock-settings.md:16]: Ideally this is auto-generated from the conda-smithy schema, but such a thing doesn't exist yet :/

File: docs/fundamentals/life-cycle.md [hunk 9]
```diff
@@ -33,10 +37,6 @@ The default label is `main`.
 When a new label is added, the package is also visible in the subchannel `<channel>/label/<label>`.
 For example, a package uploaded to conda-forge and labeled as `test` and `main` will be available in the `conda-forge` channel, but also in the `conda-forge/label/test` subchannel.
 
-As part of the CDN-driven metadata patching, packages labeled as `broken` are not included in the final repodata index.
-However, they are still available via direct URL access.
-This allows organizations to retire packages from normal, solver-driven installs without compromising the reproducibility offered by lockfiles.
-
 ## The life cycle on conda-forge
 
 The stages described above can be performed manually by any user, from their laptop.
```
[the remark this answers is not in the document]
Comment (jaimergp) [reply docs/fundamentals/life-cycle.md:42]: Any suggestions? It sounds good to me but that doesn't mean it's correct hah

File: docs/reference/glossary.md (hunk 3 above)
Comment (jaimergp) [reply docs/reference/glossary.md:2]: I only imported the ones already existing in conda-forge.org for completeness, but we are not focusing on this right now. Issue welcome :D

File: docs/reference/infrastructure/feedstocks.md (hunk 4 above)
Comment (jaimergp) [reply docs/reference/infrastructure/feedstocks.md:20]: I feel adding more details belongs in a (currently inexistent) conda-smithy documentation maybe?

File: docs/reference/infrastructure/services.md (hunk 5 above)
Comment (jaimergp) [reply docs/reference/infrastructure/services.md:1]: On second thought, tabs are not directly linkable via `#anchors`, which might hinder navigation.

File: docs/reference/pinnings.md (hunk 7 above)
Comment (jaimergp) [reply docs/reference/pinnings.md:15]: On second thought, tabs are not directly linkable via `#anchors`, which might hinder navigation. I think tabs are ok for examples and so on, but not for reference info maybe?

File: docs/reference/infrastructure/services.md (hunk 5 above)
Comment (jaimergp) [reply docs/reference/infrastructure/services.md:1]: Oh [it does support it](https://docusaurus.io/docs/markdown-features/tabs?current-os=ios#query-string) via `?query` strings but that's a bit weird :/

**Pull request merged**